recipe/meta.yaml

```diff
@@ -26,10 +26,9 @@ requirements:
 
   run:
     - python
-    - bokeh >=0.12.3
```

Comment:
Guessing we are just dropping this to avoid pull in lots of dependencies here, right?

Reply:
Yes, we'll put this into the dask metapackage instead